static/scss/header.scss

```diff
@@ -1,4 +1,5 @@
 @import '_variables';
+@import 'breakpoints';
```

Comment:
We should be able to just import `'breakpoints'` in the main file (`layout.scss`) and use it in any file we import afterwards (so we only have to have it imported in one place). We should be able to do the same thing with `_variables`.

Reply:
Fixed